src/components/DataMap/stationColor.js

```diff
@@ -58,7 +57,7 @@ function value_to_color(value, thresholds, colors) {
 }
```

Comment:
Suggest refactoring threshold and color values into single items to prevent possible errors in updating. E.g.,

```
const variableToPrecipColourScale = [
  { threshold: -87.5, color: '#8c510a' },
  ...
]
```